src/components/Fund/FundBanner.tsx

```diff
@@ -1,106 +1,161 @@
-import React from "react";
-import styles from "./FundBanner.module.css";
-import FundTabs from "../../components/FundTabs/FundTabs";
+import { Trans } from '@lingui/macro';
+import copyIcon from '../../img/icons/carbonCopyIcon.svg';
+import qrCodeIcon from '../../img/icons/carbonQrCodeIcon.svg';
+import updateNowIcon from '../../img/icons/carbonUpdateNowIcon.svg';
+import telegramIcon from '../../img/icons/telegramYellowIcon.svg';
+import twitterIcon from '../../img/icons/twitterYellowIcon.svg';
+import { numberWithCommas } from '../../data/formatting';
+import ListBox from './ListBox';
+import { NavLink } from 'react-router-dom';
+import { useState } from 'react';
 
 export default function FundBanner(props: any) {
+  const { funds, selected, setSelected } = props;
+
+  // select portfolio/trading/yield from fund manage
+  const [pty, setPty] = useState('portfolio');
```

Comment:
pty is not a good name for this variable. What if we add another tab later? we can use something like selectedFundAction.

Reply:
Yeah, this is far better, I was struggling to come up with a good name, my bad. Thanks for the suggestion.